models/docs/sources/contract_data.md

```diff
@@ -22,3 +22,23 @@ Contract can either be temporary or persistent
 {% docs balance_holder %}
 The address/account that holds the balance of the asset in contract data
 {% enddocs %}
+
+{% docs key %}
+The encoded key used to identify a specific piece of contract data. This key is typically a hashed value representing the unique identifier for the data entry.
```

Comment:
> This key is typically a hashed value representing the unique identifier for the data entry

I'm not sure if this is true. Where did you get this information from?

Reply:
Havent tested the flow via airflow @amishas157 . Was able to run the code through my local dbt setup in Dev.

Reply:
@chowbao Will update the field description per your recommendation. Tried to get these detail from referencing stellar docs and internet searches.

Reply:
> Havent tested the flow via airflow @amishas157 . Was able to run the code through my local dbt setup in Dev.

FYI you can click the `...` in a message and `Quote Reply` instead of responding in an unrelated thread. In https://github.com/stellar/stellar-dbt-public/pull/71#pullrequestreview-2210922325 case it will create a message instead of a thread because there is not code linked to create a thread 🤷‍♀️ but that's just bad github UI that we can't avoid